src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -33,11 +33,11 @@
             ]
         },
         {
-            "name": "ssmDocument_createSsmDocumentFromTemplate",
+            "name": "ssm_createDocument",
             "description": ""
```

Comment:
Please add human readable descriptions to these metrics. The intention is for people to understand what activities/data we capture when gathering usage data.